models/selectArticleComments.js

```diff
@@ -0,0 +1,28 @@
+const db = require("../db/connection");
+const { checkArticleExists } = require("../utils/checkArticleExists");
+
+exports.selectArticleComments = (article_id) => {
+  if (/[A-Za-z]/g.test(article_id)) {
+    return Promise.reject({
+      status: 400,
+      msg: "Invalid article id",
+    });
+  }
```

Comment:
This logic can be done within SQL as SQL will throw an error if you attempt to use the wrong data type in a query so would prefer that this is done by using an error handle to check the error code